src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -23,93 +22,109 @@ trait DataAccess {
   def createWorkflow(workflowDescriptor: WorkflowDescriptor,
                      workflowInputs: Traversable[SymbolStoreEntry],
                      calls: Traversable[Scope],
-                     backend: Backend): Future[Unit]
+                     backend: Backend)(implicit ec: ExecutionContext): Future[Unit]
```

Comment:
I coulda saved myself some time and used all of your implicits instead of doing it myself yesterday ;)